examtool/api/scramble.py

```diff
@@ -27,6 +27,10 @@ def scramble_group(group, substitutions, config, depth):
                 scramble_question(
                     element, [*substitutions, group_substitutions], config
                 )
+        if group.get("pick_some") == 1 and not group["title"].trim() and not group["text"].trim():
```

Comment:
Not sure about the if there is no text part. Also doesn't this not address the issue if you have a single group in another group (not one which is being picked)?

Reply:
If you've manually written
```
<group>
    <group>
    </group>
</group>
```
then you could just manually collapse them, so if you've done it that way I assume it's intentional.